python/chronos/src/bigdl/chronos/data/experimental/xshards_tsdataset.py

```diff
@@ -364,6 +364,33 @@ def _inverse_transform(df, id_col, scaler, feature_col, target_col):
                                                   self.target_col)
         return self
 
+    def unscale_xshards(self, data):
+        '''
+        Unscale the time series forecaster's numpy prediction result/ground truth.
+
+        :param data: xshards same with self.numpy_xshards.
+
+        :return: the unscaled xshardtsdataset instance.
+        '''
+        def _inverse_transform(data, scaler, scaler_index):
+            from sklearn.utils.validation import check_is_fitted
+            from bigdl.nano.utils.log4Error import invalidInputError
+
+            id = data['id'][0, 0]
+            scaler_for_this_id = scaler[id]
+            invalidInputError(not check_is_fitted(scaler_for_this_id),
+                              "scaler is not fitted. When calling scale for the first time, "
+                              "you need to set fit=True.")
+
+            return unscale_timeseries_numpy(data['y'], scaler_for_this_id, scaler_index)
```

Comment:
I think the key need to be set by users, since there are 2 possibilities: 'prediction' and 'y'